### ISSUE TITLE
Bad Contrast for Invalid Fields on Admin Gradeable in Dark Mode

### ISSUE THREAD

**Issue by eddiekrystowski** [Submitty/Submitty#6499]
**Describe the bug**
When creating/editing a gradeable in dark mode, the contrasts for editing the field is poor. Also, the invalid input is recognized by simply changing the background color, with no other indication that it is invalid which makes it less accessible.

**Expected behavior**
The contrast should be improved and the input should be marked with the invalid selector.

**To Reproduce**
1. Create a gradeable
2. Navigate to the Dates tab
3. Change TA Beta Testing Date to a date after Submission Open Date

**Configuration**
 - OS: MacOS
 - Browser: Chrome

**Screenshots**
![image](https://user-images.githubusercontent.com/28243927/119554436-a5285900-bd6a-11eb-82a2-21843ad5415d.png)
